django_project/ford3/tests/functional/test_homepage.py

```diff
@@ -1,17 +1,17 @@
-# coding=utf-8
-import unittest
+from django.test import TestCase
+from selenium import webdriver
+from selenium.webdriver.common.desired_capabilities import DesiredCapabilities
 
-from ford3.tests.functional.utils import SeleniumTestCase, selenium_flag_ready
 
+class TestHomepage(TestCase):
```

Comment:
no you should use what Rizky did. Your Test class must inherits from SeleniumTestCase.